lib/tario/compare.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 // IsSimilarHeader returns if the given headers are describing similar entries.
-func IsSimilarHeader(h *tar.Header, nh *tar.Header) (bool, error) {
+func IsSimilarHeader(h *tar.Header, nh *tar.Header, ignoreTime bool) (bool, error) {
```

Comment:
Can you change this to checkModTime and pass that in from diff command?
So later it would be easier to add diff option based on content hash.

Reply:
Sounds good for me.